harness/tests/experiment/fixtures/pytorch_onevar_model.py

```diff
@@ -0,0 +1,124 @@
+"""
+A one-variable linear model with no bias. The datset emits only pairs of (data, label) = (1, 1),
+meaning that the one weight in the model should approach 1 as gradient descent continues.
+
+We will use the mean squared error as the loss.  Since each record is the same, the "mean" part of
+mean squared error means we can analyze every batch as if were just one record.
+
+Now, we can calculate the mean squared error to ensure that we are getting the gradient we are
+expecting.
+
+let:
+    l = loss
+    w = the value of the one weight
+    R = learning rate (constant)
+
+then calculate the loss:
+
+(1)     l = (label - (data * w)) ** 2
```

Comment:
non-blocking: Better put down the initial values `w0`.

Reply:
done